src/components/Summary/style.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/index.scss';
+
+.summary {
+  display: flex;
+  align-items: center;
+  height: 100dvh;
+  justify-content: center;
+  gap: 8rem;
+
+  .profile-pic {
+    width: 25rem;
+    height: 25rem;
+    border-radius: 50%;
+    overflow: hidden;
+  }
+
+  .info {
+    display: grid;
+    justify-items: center;
+    gap: 1.25rem;
+
+    .greeting {
+      @include paragraph;
+    }
+
+    .name {
+      @include heading;
+      font-size: 3rem;
+      line-height: 2rem;
+    }
+
+    .job {
+      @include sub-heading;
+      font-size: 1.5rem;
+    }
+
+    .cta-container {
+      display: flex;
+      gap: 1rem;
+
+      .cta-dark {
+        background-color: $background_color_dark;
+        color: $light_text;
+
+        &:hover {
+          background-color: $light_text;
+          color: $background_color_dark;
+        }
+      }
+    }
+
+    .social-links {
+      display: flex;
+      gap: 1rem;
+
+      .icon {
+        width: 2rem;
+        height: 2rem;
```

Comment:
if you are giving width & height here, thn why give `width: 100%` & `height: 100%` in global CSS?

Reply:
its div of img